app/App.xaml.cs

```diff
@@ -7,6 +7,8 @@ public class App : Application
     {
         public override void Initialize()
         {
+            var x = this.GetType();
```

Comment:
`x` variables for the win.
I guess this committed by mistake?

Reply:
yes, removed